src/toM3u8.js

```diff
@@ -89,16 +116,20 @@ export const formatVttPlaylist = ({ attributes, segments }) => {
   };
 };
 
-export const organizeAudioPlaylists = playlists => {
-  return playlists.reduce((a, playlist) => {
+export const organizeAudioPlaylists = (playlists, sidxMapping) => {
```

Comment:
I know that we add a default of `{}` to `sidxMapping` in `toM3u8`, but I think it makes sense to default this to `{}` here and in the other functions where we pass it as well. I think it makes it easier to follow the default value.